multi.lic

```diff
@@ -0,0 +1,64 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#multi
+=end
+
+custom_require.call(%w[common])
+
+class Multi
+  include DRC
+
+  def initialize(args = nil)
+    if args.nil? || args.empty?
```

Comment:
❓ Would you be willing to use our arg system?

Reply:
I'll look into it more. Since they were complex arguments I didn't tackle that bringing the script over, but worth a shot. Thanks